tests/invoice_utils/test_web.py

```diff
@@ -326,3 +325,24 @@ def test_template_related_variables_with_bad_inputs_raise_errors(
 ):
     with pytest.raises(expected_error):
         http.post("/invoice", json=email_invoice_request_body)
+
+
+@pytest.mark.parametrize(
+    "environment",
+    [
+        ({"INVOICE_UTILS_INVOICE_DIR": "bad_directory"}),
+        ({"INVOICE_UTILS_DEFAULT_RULE_TEMPLATE_NAME": "bad_name.json"})
+    ],
+    indirect=["environment"]
+)
+def test_app_does_not_start_with_bad_default_rule_template(
```

Comment:
Is there a test that checks that the app doesn't start if the `DEFAULT_RULE_TEMPLATE_NAME` doesn't exist? That is to say, if `template_repo.exists(DEFAULT_RULE_TEMPLATE_NAME)` returns `False` then the `app` shouldn't start at all.

Reply:
Shouldn't the `app` fail only if the `template_repo.exists(DEFAULT_RULE_TEMPLATE_NAME)` AND then the default "basic.json" file doesn't exist?